test/dnsimple/domain_dnssec_spec.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+var testUtils = require('../testUtils');
+var dnsimple = require('../../lib/dnsimple')({
+  accessToken: testUtils.getAccessToken(),
+});
+
+const expect = require('chai').expect;
+const nock = require('nock');
+
+describe('domains', function() {
+  describe('#enableDnssec', function() {
+    var accountId = '1010';
+    var domainId = 'example.com';
+    var fixture = testUtils.fixture('enableDnssec/success.http');
+
+    it('builds the correct request', function(done) {
+      var endpoint = nock('https://api.dnsimple.com')
+        .post('/v2/1010/domains/example.com/dnssec')
+        .reply(fixture.statusCode, fixture.body);
+
+      dnsimple.domains.enableDnssec(accountId, domainId);
+
+      endpoint.done();
+      done();
+    });
+
+    it('produces an response', function(done) {
+      nock('https://api.dnsimple.com')
+        .post('/v2/1010/domains/example.com/dnssec')
+        .reply(fixture.statusCode, fixture.body);
+
+      dnsimple.domains.enableDnssec(accountId, domainId).then(function(response) {
+        var dnssec = response.data;
+        expect(dnssec.enabled).to.eq(true);
```

Comment:
I suggest use strict `expect(dnssec.enabled).to.be.true;`

source: http://chaijs.com/api/bdd/